packages/nextjs/generated/deployedContracts.ts

```diff
@@ -1,3 +1,154 @@
-const deployedContracts = null;
+const contracts = {
```

Comment:
Need to remove this file from the commit. I assume this is YourContract. It doesn't need to be included.